custom/shared/src/main/java/com/solarwinds/opentelemetry/extensions/SolarwindsSampler.java

```diff
@@ -168,9 +169,9 @@ public SamplingResult shouldSample(
   }
 
   private String constructUrl(Attributes attributes) {
-    String scheme = attributes.get(SemanticAttributes.URL_SCHEME);
-    String host = attributes.get(SemanticAttributes.SERVER_ADDRESS);
-    String target = attributes.get(SemanticAttributes.URL_PATH);
+    String scheme = attributes.get(UrlAttributes.URL_SCHEME);
+    String host = attributes.get(ServerAttributes.SERVER_ADDRESS);
+    String target = attributes.get(UrlAttributes.URL_PATH);
```

Comment:
We should update [our docs on the attributes](https://documentation.solarwinds.com/en/success_center/observability/content/configure/services/java/configure.htm#tracing-mode-per-transaction-disable-tracing-on-particular-requests) that transaction filtering regex matches against to be `url.scheme, server.address, and url.path` correct?

Reply:
It looks like what we have `scheme://host/target` is equivalent?

Reply:
I'm fretting over the part in 
> ...the request's URL (constructed with the format scheme://host/target using the Span attributes http.scheme, net.host.name, and http.target)

where we point customers to the Span attributes that they'd need to look at to figure out the value to match, those attributes should be updated to the new names... hmmm, leaving customers on older agents to scratch their heads maybe.

Reply:
oh, yeah, need updating 😅 .

Reply:
👍 made a mental note :)

Reply:
wondering whether we should remove mention of the exact attributes(implementation detail) because it looks like the structure of URL presented should be enough?

Reply:
yes, or we can link to the HTTP Server semconv instead. it could help the customer understand what is actually captured for their URL so they can tailor the regex to the attribute values.